src/cockpit/packages.py

```diff
@@ -146,7 +170,7 @@ def filename_variants(filename, locale):
                 yield f'{base}.{locale}.{ext}.gz'
 
                 if '_' in locale:
-                    language, _, _ = locale.partition(' ')
+                    language, _, _ = locale.partition('_')
```

Comment:
Is that really right? It looks correct for most locales like `de`, but we need to retain the country for at least en, nb, pt, and zh. In general, we should prefer `ll_CC.po` if we have it, and if not fall back to to `ll.po`.

Also, personally I prefer `language = locale.split('_')[0]`, but no strong opinion.

Reply:
Look up to the line just before the `if`.  We check the full locale name first.  The stripped down version comes next.

Reply:
I see, thanks! So @jelly , please feel free to resolve this or replace the partition with a simpler split()[0].